### PR TITLE
Refactored to use Upstash

### PR DESCRIPTION
**Pull request by steven-tey**
### Description

Refactors demo to use Upstash instead of Edge Config.

### Demo URL

https://cron-template.vercel.app/

### Type of Change

- [ ] New Example
- [ ] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)

### New Example Checklist

- [ ] 🛫 `npm run new-example` was used to create the example
- [ ] 📚 The template wasn't used but I carefuly read the [Adding a new example](https://github.com/vercel/examples#adding-a-new-example) steps and implemented them in the example
- [ ] 📱 Is it responsive? Are mobile and tablets considered?

### CODE REVIEW

File: solutions/cron/README.md [hunk 1]
```diff
@@ -26,24 +27,11 @@ https://cron-template.vercel.app/
 
 You can choose from one of the following two methods to use this repository:
 
-### 1. Set Up Environment variables
```
Comment (manovotny) [inline solutions/cron/README.md:29]: Isn't this still needed? Environment variables are used for Upstash.

Comment (manovotny) [reply solutions/cron/README.md:29]: Or is this because it's using an integration now?

**Review by dferber90 — APPROVED** [review]

**Review by leerob — APPROVED** [review]

File: solutions/cron/README.md (hunk 1 above)
Comment (leerob) [reply solutions/cron/README.md:29]: Yep, the integration does this!

File: solutions/cron/.env.example [hunk 2]
```diff
@@ -1,4 +1,2 @@
-EDGE_CONFIG_ID=
-EDGE_CONFIG=
-TEAM_ID_VERCEL=
-ACCESS_TOKEN_VERCEL=
\ No newline at end of file
+UPSTASH_REDIS_REST_URL=
```
Comment (gregegan) [inline solutions/cron/.env.example:1]: i'm looking into this CRON project, and i'm quite confused what the upstash env variables are for - they don't look to be used anywhere? And looks to be complaining now for the KV env variables instead @steven-tey

**Pull request merged**